etcdserver/server.go

```diff
@@ -602,7 +602,47 @@ func (s *EtcdServer) LeaderStats() []byte {
 
 func (s *EtcdServer) StoreStats() []byte { return s.store.JsonStats() }
 
+func (s *EtcdServer) isReadyToAddNewMember() bool {
+	if !s.cfg.StrictReconfigCheck {
+		// the option -strict-reconfig-check isn't passed to etcd,
+		// allow adding a new member unconditionally
+		return true
+	}
+
+	nmembers := 1
+	nstarted := 0
+
+	for _, member := range s.cluster.members {
+		if len(member.Name) != 0 {
```

Comment:
Ah, sorry, I missed your intention. I'll add a member function for checking started or not to Member in the next update.

BTW, I made isReadyToAddNewMember() a member function. Is this ok?